docs/dev/building-on-zksync/contracts/differences-with-ethereum.md

```diff
@@ -318,6 +318,16 @@ No actions are required from your side as all the calls to the precompiles are d
 
 In contrast to Ethereum, zkSync Era ecrecover always return a zero address for the zero digests. Be careful with adapting crypto primitives that rely on that, specifically, it affects [secp256k1 mul verification via ecrecover](https://ethresear.ch/t/you-can-kinda-abuse-ecrecover-to-do-ecmul-in-secp256k1-today/2384).
 
+## Native AA vs EIP 4337
+
+The native account abstraction of zkSync and Ethereum's EIP 4337 aim to enhance accounts' flexibility and user experience, but they differ in critical aspects listed below:
+
+1. **Implementation Level**: zkSync's account abstraction is integrated at the protocol level, providing a unified transaction flow. On the other hand, EIP 4337 introduces a separate transaction flow on top of Ethereum's existing one.
+2. **Account Types**: zkSync introduces Smart Contract Accounts and Paymasters. Smart Contract Accounts are fully programmable, offering a wide range of customizations. Paymasters sponsor transactions, allowing users to pay fees in ERC20 tokens. EIP 4337, however, does not introduce a concept equivalent to Paymasters.
```

Comment:
4337 does mention paymasters - https://eips.ethereum.org/EIPS/eip-4337#extension-paymasters

Reply:
added that paymaster isn't supported for EOAs.